VSRAD.Syntax/Options/GeneralOptions.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System.Linq;
+using System.Text;
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.VisualStudio.Settings;
+using System;
+using Task = System.Threading.Tasks.Task;
+
+namespace VSRAD.Syntax.Options
+{
+    public enum SortState
+    {
+        [Description("by line number")]
+        ByLine = 1,
+        [Description("by line number descending")]
+        ByLineDescending = 2,
+        [Description("by name")]
+        ByName = 3,
+        [Description("by name descending")]
+        ByNameDescending = 4,
+    }
+
+    public class GeneralOptions : BaseOptionModel<GeneralOptions>
+    {
+        private const string InstructionCollectionName = "RadeonAsmInstructionCollection";
+        private static readonly Regex fileExtensionRegular = new Regex(@"^\.\w+$");
```

Comment:
Code style consistency: rename private member to `_fileExtensionRegular`?

Reply:
Thank you, fixed in 024da88